src/earn/EarnEnterAmount.tsx

```diff
@@ -605,15 +608,40 @@ function FeeDetailsBottomSheet({
   feeCurrency,
   estimatedFeeAmount,
   maxFeeAmount,
+  swapTransaction,
+  pool,
+  token,
 }: {
   forwardedRef: React.RefObject<BottomSheetModalRefType>
   title: string
   testID: string
   feeCurrency?: TokenBalance
   estimatedFeeAmount?: BigNumber
   maxFeeAmount?: BigNumber
+  swapTransaction?: SwapTransaction | undefined
+  pool: EarnPosition
+  token: TokenBalance
 }) {
   const { t } = useTranslation()
+  const depositToken = useTokenInfo(pool.dataProps.depositTokenId)
+
+  if (!depositToken) {
+    // should never happen
+    throw new Error(`Token info not found for token ID ${pool.dataProps.depositTokenId}`)
+  }
+
+  const swapFeeAmount = useMemo(() => {
+    if (swapTransaction && swapTransaction.appFeePercentageIncludedInPrice) {
+      return new BigNumber(swapTransaction.buyAmount)
+        .multipliedBy(new BigNumber(swapTransaction.appFeePercentageIncludedInPrice).shiftedBy(-2)) // To convert from percentage to decimal
+        .shiftedBy(-depositToken.decimals)
```

Comment:
the app fee is a % of the from amount, not the to amount. So this should be using the amount the user inputs

Reply:
Fixed in https://github.com/valora-inc/wallet/commit/fff206b091c9f59a4a2c2ea84e6cf6e7a7469de2!